commit/core.pyx

```diff
@@ -726,6 +728,53 @@ cdef class Evaluation :
             ERROR( 'Threads not set; call "set_threads()" first' )
         if self.A is None :
             ERROR( 'Operator not built; call "build_operator()" first' )
+        
+        # Confidence map.
+        self.confidence_map_img = None
+        self.set_config('confidence_map_filename', None)
+        self.set_config('confidence_map_dim', None)
+        self.set_config('confidence_map_pixdim', None)
```

Comment:
Do we need these values? The confidence is on the voxels, so we need to enforce that the resolution of the map matches the one of the dMRI data

Reply:
No, they are not needed.